src/test_util.rs

```diff
@@ -0,0 +1,54 @@
+use std::{cmp::PartialEq, fmt::Debug};
```

Comment:
Maybe this file should reside within `tests` subdirectory in the root?

Reply:
ok, I moved now all tests into the integrations tests folder.